### PR TITLE
Avoid 'Taxonomy has already been taken'. Explicitly cast rhel7_os to str so it won't fail being converted to JSON.

### PR DESCRIPTION
**Pull request by lhellebr**

### CODE REVIEW

File: robottelo/constants/__init__.py [hunk 1]
```diff
@@ -1727,7 +1727,7 @@
 ]
 
 VMWARE_CONSTANTS = {
-    'cluster': 'Satellite_Engineering',
+    'cluster': 'Satellite-Engineering',
```
Comment (lhellebr) [inline robottelo/constants/__init__.py:1730]: These "constants" should be variables IMO but let's leave it for another PR.

File: robottelo/api/utils.py [hunk 2]
```diff
@@ -293,7 +293,7 @@ def configure_provisioning(org=None, loc=None, compute=False, os=None):
     # Create a Product, Repository for custom RHEL6 contents
     product = entities.Product(organization=org).create()
     repo = entities.Repository(
-        product=product, url=settings.rhel7_os, download_policy='immediate'
+        product=product, url=str(settings.rhel7_os), download_policy='immediate'
```
Comment (mirekdlugosz) [inline robottelo/api/utils.py:296]: I think it would be better to add `rhel7_os` to wrapper exceptions list in [robottelo/config/facade.py](https://github.com/SatelliteQE/robottelo/blob/master/robottelo/config/facade.py). I see `rhel8_os` is already there.

**Review by pondrejk — APPROVED** [review]
ack pending Mirek's comment

**Comment by lhellebr** [comment]
Mirek's comment addressed and it indeed fixes the issue:

```
$ pytest tests/foreman/longrun/test_provisioning_computeresource.py
============================================================ test session starts ============================================================
platform linux -- Python 3.8.6, pytest-5.4.3, py-1.9.0, pluggy-0.13.1
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/lhellebr/git/robottelo
plugins: services-1.3.1, ibutsu-1.1.1, xdist-1.34.0, cov-2.10.0, mock-3.3.1, forked-1.2.0
collecting ... 2021-01-13 11:11:26 - conftest - DEBUG - Collected 2 test cases
collected 2 items                                                                                                                           

tests/foreman/longrun/test_provisioning_computeresource.py ..                                                                         [100%]

======================================================= 2 passed in 599.98s (0:09:59) =======================================================
```
